contracts/native-router/src/contract.rs

```diff
@@ -167,23 +167,29 @@ fn try_route_balances(deps: DepsMut, env: Env) -> Result<Response, ContractError
             // neutron fees for
             let count = Uint128::from(1 + denom_balances.len() as u128);
 
-            for mut coin in denom_balances {
-                // if we distribute neutron, we need to keep a
-                // reserve for ibc gas costs.
-                // this is safe because we pass target denoms.
-                if coin.denom == "untrn" {
+            for coin in denom_balances {
+                // non-neutron coins get distributed entirely
```

Comment:
any reason for bringing this back? or could it be an old rebase?

Reply:
My bad, fixed.